service/awsconfig/v10/ebs/mock.go

```diff
@@ -9,13 +9,14 @@ import (
 )
 
 type EC2ClientMock struct {
-	customObject v1alpha1.AWSConfig
-	ebsVolumes   []EBSVolumeMock
+	CustomObject v1alpha1.AWSConfig
+	EBSVolumes   []EBSVolumeMock
```

Comment:
The mock is wrong because its used in the ebsvolume resource which is why everything is exported. I'll sort this out if this actually works!

Reply:
I stumbled upon this mocking technique in several places. I feel like interfaces are defined in various places and cross used everywhere, like there is not a real guideline on how to do this. Maybe we can agree on something in the long run and get this a bit more straight.

Reply:
I like this mocking approach and find it very useful. I agree we should try and standardise this a bit and write it up in fmt too.

In this case I exported the mock because I just wanted to confirm the tests still work. I'll clean this up later.